apollo-federation/src/schema/position.rs

```diff
@@ -188,6 +188,15 @@ impl TypeDefinitionPosition {
         )
     }
 
+    pub(crate) fn as_composite_type(&self) -> Option<CompositeTypeDefinitionPosition> {
```

Comment:
we already have the `TryFrom<TypeDefinitionPosition> for CompositeTypeDefinitionPosition` trait implemented so I think this is redundant?